Tasks/Gradle/CodeAnalysis/Common/BaseTool.ts

```diff
@@ -78,6 +78,21 @@ export abstract class BaseTool implements IAnalysisTool {
         return tl.getBoolInput(this.uiInputName, false);
     }
 
+    protected findHtmlReport(xmlReport: string): string {
```

Comment:
This is a good point. FindBugs on Gradle does not produce HTML reports unless specified, but if it does it is "main.html".